app/helpers/houston/dashboards/staging_helper.rb

```diff
@@ -2,8 +2,11 @@ module Houston::Dashboards
   module StagingHelper
 
     def checkboxes(pull_request)
-      checked = completed_checkboxes(pull_request)
-      total = total_checkboxes(pull_request)
+      checked, total = 0, 0
+      if pull_request.body
+        checked, total = completed_checkboxes(pull_request), total_checkboxes(pull_request)
+      end
+
```

Comment:
We're passing `pull_request` to `completed_checkboxes` and `total_checkboxes`. Why does the calling code have to check `pull_request.body`? How does it know that we care about `.body`? It should be totally naive about the implementation details of `completed_checkboxes` and `total_checkboxes`.

Also, no need to add an `if` statement. You can just do:

```diff
     def completed_checkboxes(pull_request)
-      pull_request.body.scan("[x]").count
+      pull_request.body.to_s.scan("[x]").count
     end
```

— which says `pull_request.body`, whatever you are (_I_ certainly don't check types), I need you to be a String; so do that.

That changes makes this code more confident, [as Avdi would say](http://www.confidentruby.com/).